inc/class-my-yoast-api-request.php

```diff
@@ -310,4 +334,17 @@ protected function remove_access_token( $user_id ) {
 		// Remove the access token entirely.
 		$this->get_client()->remove_access_token( $user_id );
 	}
+
+	/**
+	 * Retrieves the installed addons as http headers.
+	 *
+	 * @codeCoverageIgnore
+	 *
+	 * @return array The installed addon versions.
+	 */
+	protected function get_installed_addons_as_headers() {
```

Comment:
shouldn't the `-version` suffix also be added in this function, because they are after all the actual header names. The function name implies it return the actual headers.

Reply:
Decided to rename the method. I have chosen to not do the formatting here to follow the single responsibility principle